solutions/devsprint-julio-fernandes-2/FinanceAppTests/Service/Entities/HomeDataUnitTests.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  HomeDataUnitTests.swift
+//  FinanceAppTests
+//
+//  Created by Gabriela Sillis on 21/06/22.
+//
+
+import XCTest
+@testable import FinanceApp
+
+final class HomeDataUnitTests: XCTestCase {
+    let activity = Activity(name: "any_name", price: 80, time: "18000")
```

Comment:
Considere sempre deixar o seu objeto em que está realizando testes com o nome de sut (system under test)!

```suggestion
    private let sut = Activity(name: "any_name", price: 80, time: "18000")
```

Reply:
feito